networker/networker.go

```diff
@@ -267,6 +261,13 @@ func (netw *Combined) start(
 
 	netw.publisher.Publish("starting vpn")
 
+	// Always disable IPv6 with sysctl in the system
+	// We also do that when there is a change in network interfaces
+	if err = netw.ipv6Blocker.Block(); err != nil {
+		log.Println(internal.ErrorPrefix, "Failed to block ipv6 using sysctl ", err)
```

Comment:
Right now there are three blocks of code with the same log message if blocking fails. I think adding more context to the log message would help with debugging in the future